playground/backend/internal/utils/preparators_utils.go

```diff
@@ -36,3 +37,16 @@ func GetPreparators(sdk pb.Sdk, filepath string) (*[]preparators.Preparator, err
 	}
 	return prep, nil
 }
+
+//SpacesToEqualsOption prepares pipelineOptions by replacing spaces to equals
+func SpacesToEqualsOption(pipelineOptions string) string {
+	preparedOptions := strings.Split(pipelineOptions, " ")
+	for index, str := range preparedOptions {
+		if index%2 == 0 && index != 0 {
+			preparedOptions[index] = fmt.Sprintf(" %s", str)
+		} else if index%2 != 0 {
+			preparedOptions[index] = fmt.Sprintf("=%s", str)
+		}
+	}
```

Comment:
This logic is a bit implicit, let's add a comment that explains what is happening here

Reply:
Done.

Reply:
Done.

Reply:
Could we rename this function to `ReplaceSpacesWithEquals` ?